SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -300,7 +283,11 @@ void ASpatialDebugger::LoadIcons()
 
 void ASpatialDebugger::OnEntityAdded(const Worker_EntityId EntityId)
 {
-	check(NetDriver != nullptr && !NetDriver->IsServer());
+	check(NetDriver != nullptr);
+	if (NetDriver->IsServer())
```

Comment:
These seem easier and cheaper to check at calling sight, but I guess this is gonna be reworked anyway

Reply:
yes this has to be rejigged soonish anyway